product-service/getProductById.ts

```diff
@@ -0,0 +1,40 @@
+import { APIGatewayProxyHandler } from 'aws-lambda';
+import 'source-map-support/register';
+
+import products from './db/productList.json';
+
+export const handler: APIGatewayProxyHandler = async (event) => {
+  const id = event.pathParameters?.id;
+  if (!id) {
+    return {
+      statusCode: 404,
+      body: 'Error: No id',
+      headers: {
+        'Access-Control-Allow-Origin': '*',
+        'Access-Control-Allow-Credentials': true,
+      }
+    };
+  }
+
+  const product = products.find((item) => item.id === id);
+
+  if (!product) {
+    return {
+      statusCode: 400,
+      body: `No any product with this id: ${id} `,
+      headers: {
+        'Access-Control-Allow-Origin': '*',
+        'Access-Control-Allow-Credentials': true,
+      }
+    };
+  }
+
+  return {
+    statusCode: 200,
```

Comment:
You may create util generateApiReponse for both handlers with the same headers,etc.